GameServerLib/GameObjects/AttackableUnits/AI/Champion.cs

```diff
@@ -167,6 +167,8 @@ public bool CanMove()
         public void UpdateMoveOrder(MoveOrder order)
         {
             MoveOrder = order;
+
+            ApiEventManager.OnChampionDamageTaken.Publish(this);
```

Comment:
Um, you move and you trigger damage taken event? Okay.

Reply:
Resolved and added Unit and Champion crowd control event handlers